docs/Tutorials/Quick-Start.md

```diff
@@ -148,19 +143,31 @@ Products Controller
 class ProductsController extends ApController {
 	// Upload an image
 	public function upload($productId = null) {
-		if (!$this->request->is('get')) {
-			if ($this->Products->ProductImages->upload($productId, $this->request->data)) {
-				$this->Session->set(__('Upload successful!');
-			}
-		}
+
+            $this->loadModel("ProductImages");
```

Comment:
Perhaps i am not yet well versed with cakePHP or my Configs are different or cakePHP has changed. The association wont  load if  the model is not loaded.